transifex.sh

```diff
@@ -18,23 +26,23 @@ index 89663f7..ddf5d23 100644
 --- b/fr-FR/Collabora-Office.adml
 +++ a/fr-FR/Collabora-Office.adml
 @@ -1,5 +1,7 @@
- <?xml version="1.0" encoding="utf-8"?>

```

Comment:
This hunk and the one below changes the line ending of the embedded patch. It's not good.